attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -98,10 +111,12 @@ impl Attester for TdxAttester {
     }
 
     async fn check_init_data(&self, init_data: &[u8]) -> Result<()> {
+        use crate::utils::pad;
+        use scroll::Pread;
```

Comment:
Is this change related to the PR?

Reply:
it was needed when I had the `tdx-attest-rs` disabled but I can restore.

Reply:
fixed